pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -233,7 +317,11 @@ func (db *DB) GetBandwidthAllocations() (map[storj.NodeID][]*Agreement, error) {
 		rbaBytes := []byte{}
 		agreement := &Agreement{}
 		var satellite []byte
-		err := rows.Scan(&satellite, &rbaBytes, &agreement.Signature)
+		var uplink []byte
+		var serialnum string
+		var total, max_size, createdUnixSec, expirationUnixSec, startofthedayUnixSec int64
+		var action string
+		err := rows.Scan(&satellite, &rbaBytes, &agreement.Signature, &uplink, &serialnum, &total, &max_size, &createdUnixSec, &expirationUnixSec, &action, &startofthedayUnixSec)
```

Comment:
Limit columns in SELECT to avoid many unused variables.

Reply:
updated